src/command.rs

```diff
@@ -337,12 +367,19 @@ impl Command for Nu {
 
         #[cfg(feature = "plugin")]
         {
-            signature = signature.named(
-                "plugin-config",
-                SyntaxShape::String,
-                "start with an alternate plugin cache file",
-                None,
-            );
+            signature = signature
+                .named(
+                    "plugin-config",
+                    SyntaxShape::String,
+                    "start with an alternate plugin cache file",
+                    None,
+                )
+                .named(
+                    "plugins",
+                    SyntaxShape::List(Box::new(SyntaxShape::String)),
+                    "list of plugin executables to load, separately from the cache file",
```

Comment:
Does this list contain the the name (inc), the filename (nu_plugin_inc), or the command name (inc)? As I mentioned with your PR that I landed earlier today, we should standardize because it's a bit confusing.

Reply:
This is executable file names. It's meant to be used when adding to a plugin cache isn't convenient.

Reply:
I think I've cleared this up so that it appears as a filepath now, and says 'executable files'